src/styles/blocks/card.scss

```diff
@@ -0,0 +1,102 @@
+.card {
+  display: flex;
+  margin: 10px;
+  width: 200px;
+  flex-direction: column;
+  background-color: $card-color;
+  box-sizing: border-box;
+  border: 1px solid $border-color;
+  border-radius: 5px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 0;
+  }
+
+  &__title {
+    text-align: left;
+    color: $main-text-color;
+    font-family: Roboto, sans-serif;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    display: flex;
```

Comment:
It's not clear why `.card__title` needs to be a flex container. Since it only contains text, there's no need for flex layout properties here.